tests/test_alias.py

```diff
@@ -2,14 +2,11 @@
 #
 # SPDX-License-Identifier: Apache-2.0
 
-import os
 import subprocess
 
 import pytest
 from conftest import cmd
 
-assert 'TOXTEMPDIR' in os.environ, "you must run these tests using tox"
```

Comment:
Ha, I'm finally understanding the "detox" pun :-)

This would be great: because of too many layers, so far my only solution to run a test in a debugger was `PUDB_TTY`. It worked... only most of the time!

- https://github.com/inducer/pudb/issues/598#issuecomment-1839292342

Don't forget to update b7e091d2e902

Reply:
Unless... the `uv` replacement merely substitutes another layer of indirection, still getting in the way and still making it hard to run a test in a debugger directly?

Sorry for thinking out loud. Hey it's just a draft PR ;-)

Reply:
I want to reduce the layering too, it should be possible to run `pytest` directly IMO. `uv` is an optional utility, but I do want to make it the default for running in CI.